packages/rulesets/src/spectral/test/xms-pageable-for-get-list-calls.test.ts

```diff
@@ -0,0 +1,193 @@
+import { Spectral } from "@stoplight/spectral-core"
+import linterForRule from "./utils"
+
+let linter: Spectral
+
+const errorMessage = "For all LIST APIs (a.k.a collection GETs), it is important to include the `x-ms-pageable` property."
+
+beforeAll(async () => {
+  linter = await linterForRule("XmsPageableForGetListCalls")
+  return linter
+})
+
+test("XmsPageableForGetListCalls should find errors if x-ms-pagebale property is not defined.", () => {
+  const oasDoc = {
+    swagger: "2.0",
+    paths: {
+      "/{resourceUri}/providers/Microsoft.ConnectedVMwarevSphere/virtualMachine": {
```

Comment:
>/{resourceUri}/providers/Microsoft.ConnectedVMwarevSphere/virtualMachine [](http://example.com/codeflow?start=7&length=72)

can we even add examples for regular and nested resourceTypes in addition to the extensionResourceTypes

Reply:
I already added on line 38